packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -611,20 +611,30 @@ export enum InstanceClass {
    */
   STORAGE_COMPUTE_1 = 'storage-compute-1',
 
+  /**
+   * Storage/compute balanced instances, 1st generation
+   */
+  H1 = 'h1',
+
   /**
    * High performance computing powered by AWS Trainium
    */
-  TRN1 = 'trn1',
+  TRAINING_ACCELERATOR1 = 'training-accelerator1',
```

Comment:
Where are these descriptions pulled from? Some of them seem mismatched after the changes, or were they incorrect before?

Reply:
Neither `trn1` nor `trn1n` had a symbolic "alias" given to them when they were added, unlike every other instance class. I added one, didn't get it from anywhere really, looked up the AWS docs and tried to match the other's.